src/vuestic-theme/vuestic-components/va-rating/VaRatingItem.demo.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <VbDemo>
+    <VbContainer title="default">
```

Comment:
Demo should cover emits.

Reply:
@asvae but this component doesn't have logic without `VaRating` component

Reply:
Yes it does.